Classes/Emogrifier.php

```diff
@@ -952,6 +939,44 @@ private function removeInvisibleNodes(\DOMXPath $xPath)
     }
 
     /**
+     * Before be begin processing the CSS file, parse the document and normalize all existing CSS attributes.
+     * This changes 'DISPLAY: none' to 'display: none'.
+     * We wouldn't have to do this if DOMXPath supported XPath 2.0.
+     * Also store a reference of nodes with existing inline styles so we don't overwrite them.
+     *
+     * @param \DOMXPath $xPath
+     *
+     * @return void
+     *
+     * @throws \RuntimeException
+     */
+    private function normalizeStyleAttributesOfAllNodes(\DOMXPath $xPath)
+    {
+        try {
+            $nodesWithStyleAttribute = $this->getAllNodesWithStyleAttribute($xPath);
+            /** @var \DOMElement $node */
+            foreach ($nodesWithStyleAttribute as $node) {
+                if ($this->isInlineStyleAttributesParsingEnabled) {
+                    $this->normalizeStyleAttributes($node);
+                }
+                // Remove style attribute in every case, so we can add them back (if inline style attributes
+                // parsing is enabled) to the end of the style list, thus keeping the right priority of CSS rules;
+                // else original inline style rules may remain at the beginning of the final inline style definition
+                // of a node, which may give not the desired results
+                $node->removeAttribute('style');
+            }
+        } catch (\RuntimeException $e) {
+            if ($this->debug) {
+                throw new \RuntimeException(
```

Comment:
We can just rethrow `$e` here, I think.

Reply:
Same reason as above, created new Exception object to be able to define a different exception code just for this method.
Do you have any suggestions instead on how to do it better? Thanks!